plugins/generator-1.17.1/forge-1.17.1/procedures/spawn_entity.java.ftl

```diff
@@ -0,0 +1,16 @@
+<#assign entity = generator.map(field$entity, "entities", 1)!"null">
+<#if entity != "null">
+if(world instanceof ServerLevel _level) {
+	<#if !entity.toString().contains(".CustomEntity")>
+	Entity entityToSpawn = new ${generator.map(field$entity, "entities", 0)}(${entity}, _level);
+	<#else>
+	Entity entityToSpawn = new ${entity}(${entity.toString().replace(".CustomEntity", "")}.entity, _level);

```

Comment:
You can already change this part to how it will be now, which most likely means

`@JavaModNameEntityTypes.@REGISTRYNAME`

same for other procedures that do this